packages/cli/src/services/password.service.ts

```diff
@@ -0,0 +1,45 @@
+import { BadRequestError } from '@/errors/response-errors/bad-request.error';
+import { Service } from 'typedi';
+import { compare, genSaltSync, hash } from 'bcryptjs';
+import {
+	MAX_PASSWORD_CHAR_LENGTH as maxLength,
+	MIN_PASSWORD_CHAR_LENGTH as minLength,
+} from '@/constants';
+
+@Service()
+export class PasswordService {
```

Comment:
I think the issue here is that typedi uses the word `Service` to describe everything. If we were to have a decorator called `Injectable`, this would be less confusing.
Regarding this class itself: making this injectable means that 
1. it can use the existing mechanism to be injected wherever it's needed
2. it can be easily mocked using `jest-mock-extended` (which uses typed proxies), instead of forcing us to use `jest.mock` which is slower, and will block us even more from ever migrating from CJS to ESM. in general, we should be getting rid of `jest.mock` and any code patterns that force us to use `jest.mock`.

That said, maybe we can already alias the `Service` decorator as `Injectable` and start adding `Helper` suffix instead of `Service` to helper classes.